app/lib/jsonapi.js

```diff
@@ -510,7 +529,10 @@ jsonapi.routeModelPostObjectResource = function(app, Model) {
  * @param {function} done - Express route callback expecting req and res as parameters
  */
 jsonapi.routeModelResource = function(app, model, method, path, done) {
-  if (!model.jsonapi || !model.jsonapi[method]) { return; }
+  debug('jsonapi.routeModelResource -- method = %s, path = %s', method, path);
+  if (!model.jsonapi || !model.jsonapi[method]) {
```

Comment:
Please integrate this rule to enforce line breaks before`if` statements such as here: https://eslint.org/docs/rules/padding-line-between-statements